petab/v2/core.py

```diff
@@ -929,6 +950,37 @@ def _validate(self) -> Self:
 
         return self
 
+    @property
+    def prior_dist(self) -> Distribution:
+        """Get the pior distribution of the parameter."""
+        if self.estimate is False:
+            raise ValueError(f"Parameter `{self.id}' is not estimated.")
+
+        if self.prior_distribution is None:
+            return Uniform(self.lb, self.ub)
+
+        if not (cls := _prior_to_cls.get(self.prior_distribution)):
+            raise ValueError(
+                f"Prior distribution `{self.prior_distribution}' not "
+                "supported."
+            )
+
+        if str(self.prior_distribution).startswith("log-"):
+            log = True
+        elif str(self.prior_distribution).startswith("log10-"):
+            log = 10
+        else:
+            log = False
+
+        if cls == Exponential:
+            if log is not False:
+                raise ValueError(
+                    "Exponential distribution does not support log "
+                    "transformation."
+                )
+            return cls(*self.prior_parameters, trunc=[self.lb, self.ub])
```

Comment:
Out of curiosity, why explicitly Exponential but not the other distributions for which log- is not supported?

Reply:
In `distributions.py`, I enabled log-versions for all distributions for which I found evidence that they were used anywhere (even if not supported as PEtab prior distribution). Exponential was the only one for which this wasn't the case. Therefore, this is the only one that does not support the `log` argument.

Reply:
I think it's closely related to the discussion of whether parameter bounds should truncate the prior distributions, where we concluded that they should. Fixing a parameter is effectively changing the bounds to lb=ub.

Reply:
Alright, fair enough! Perhaps add a comment then, since it differs from the spec